linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_networkconfig_modification/tests/augen_correct_rules_watch_rules_without_key.pass.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# packages = audit
+
+echo "-a always,exit -F arch=b32 -S sethostname,setdomainname -F key=audit_rules_networkconfig_modification" >> /etc/audit/rules.d/networkconfig.rules
+echo "-a always,exit -F arch=b64 -S sethostname,setdomainname -F key=audit_rules_networkconfig_modification" >> /etc/audit/rules.d/networkconfig.rules
```

Comment:
Is this correct?
These audit rules are still adding a key.

Reply:
Yes, this rule is a bit special because it combines syscalls and watches. And I did not want to modify other rules than the ones dealing with watch rules. I know it is not perfect, but I would propose that as a a separate PR.